src/services/services.ts

```diff
@@ -3282,7 +3282,9 @@ export function getPropertySymbolsFromContextualType(node: ObjectLiteralElementW
         // Bad discriminant -- do again without discriminating
         return mapDefined(contextualType.types, t => t.getProperty(name));
     }
-    return discriminatedPropertySymbols;
+    // by eliminating duplicates we might even end up with a single symbol
+    // that helps with displaying better quick infos on properties of union types
+    return Array.from(new Set(discriminatedPropertySymbols));

```

Comment:
I think we technically have a `dedupe` helper but this is probably fine.

Reply:
oh, cool! I was looking for one but couldn't find it 😅 I'll switch to it in a sec

Reply:
(we technically _also_ have `arrayFrom` which is faster than `Array.from` but whatever)